package.json

```diff
@@ -32,7 +33,6 @@
     "react-native-screens": "^2.2.0",
     "react-native-snackbar": "^2.0.4",
     "react-native-svg": "^9.13.6",
-    "react-native-ui-kitten": "^4.4.1",
```

Comment:
@artyorsh Some times I am so confused about using  `react-native-ui-kitten `or `@ui-kitten/components`. What is the difference between these two?. I didn't see anything about this in doc.

Reply:
It is same packages. We deprecated `react-native-ui-kitten` when moved to monorepo, because there are a lot of additional modules coming with it. Just for consistency

Reply:
Ok